wrangler-core/src/main/java/io/cdap/wrangler/parser/MigrateToV2.java

```diff
@@ -56,7 +56,7 @@ public MigrateToV2(@Nullable String recipe, String delimiter) {
   }
 
   public MigrateToV2(String recipe) {
-    this(recipe, "\n");
+    this(recipe, "\\r?\\n");
```

Comment:
I've seen recipes with `\r\n` in the past and they seemed to work. Were only some directives stripping the `\r` or something?